plugins/com.google.cloud.tools.eclipse.integration.appengine/src/com/google/cloud/tools/eclipse/integration/appengine/ProjectContextMenuTest.java

```diff
@@ -96,24 +77,6 @@ public void testDynamicWebProjectJava8() {
         not(MenuMatcher.hasMenuItem("Reconfigure for App Engine Java 8 runtime")));
   }
 
-  @Test
-  public void testAppEngineStandardJava7() {
```

Comment:
This test should be kept until we remove App Engine Java 7

Reply:
done